src/main/java/org/example/homework6/Calculator.java

```diff
@@ -0,0 +1,77 @@
+package org.example.homework6;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Calculator {
+    String decisionValue;
+    Scanner scanner = new Scanner(System.in);
+
+    public void calculator() {
+        double firstNum;
+        double secondNum;
+
+        do {
+            System.out.print("Enter a number:");
+            try {
+                firstNum = scanner.nextDouble();
+                scanner.nextLine();
+            } catch (InputMismatchException e) {
+                System.out.println("Invalid value! Number required.");
+                scanner.nextLine();
+                continue;
+            }
+
+            System.out.print("Enter an operation (+, -, *, /):");
+            String arithmeticOpr = scanner.nextLine();
+
+            System.out.print("Enter another number:");
+            try {
+                secondNum = scanner.nextDouble();
+                scanner.nextLine();
+            } catch (InputMismatchException e) {
+                System.out.println("Invalid value! Number required.");
+                scanner.nextLine();
+                continue;
+            }
+
+            if ("/".equals(arithmeticOpr) && secondNum == 0) {
+                System.out.println("Error! division by 0");
+                continue;
+            }
+
+            switch (arithmeticOpr) {
```

Comment:
consider extracting switch to a separate method. e.g. 
`private double calculateResult(String operator, double num1, double num2)`